server_src/lib/dataFunctions/historyClean.js

```diff
@@ -762,7 +762,7 @@ class HistoryClean {
     }
   }
 }
-HistoryClean.start();
+
```

Comment:
We should check that this is is working. It was moved to the bottom of `index.js`, but to be honest it resides better in `app-core.js` as a special startup-task function.